cnd/src/config.rs

```diff
@@ -372,21 +381,46 @@ mod tests {
         let file_contents = vec![
             r#"
             listen = ["/ip4/0.0.0.0/tcp/9939"]
+            peers = [ { id = "QmfUfpC2frwFvcDzpspnfZitHt5wct6n4kpG5jzgRdsxkY", address_hints = [ "/ip4/1.1.1.1/tcp/9939" ] } ]
```

Comment:
This structure is quite verbose.

Do we need the ability to provide multiple addresses for the same peer?
Do we even need to provide the PeerId? Asked differently, do we care about the validation that exactly the peer with this ID is behind the provided address?

Libp2p actually has a way of encoding the peerId of an address in a multi-address: https://docs.libp2p.io/concepts/peer-id/#peerids-in-multiaddrs

Maybe we don't even have to parse that manually and can just hand it off to libp2p?

Reply:
We only need the ip for our purpose so I'll simplify the config file :]